spec/models/purchase_spec.rb

```diff
@@ -4,15 +4,15 @@
   before(:each) do
```

Comment:
- [ ] As requested of you by the previous reviewer, when I run ```rspec``` to check the unit tests in your spec folder, they are all failing. Kindly look into why this is the case.

![image](https://user-images.githubusercontent.com/86318284/208608763-cbeb4fd3-b1bd-4844-9f55-cb46bd1cf0e4.png)